specificatie/genereervariant/openapi.yaml

```diff
@@ -519,6 +508,162 @@ paths:
                   to a temporary overloading or maintenance of the server.
                 instance: https://datapunt.voorbeeldgemeente.nl/api/v1/resourcenaam?parameter=waarde
                 code: notAvailable
+  /kadastraalonroerendezaken/burgerservicenummer:
+    post:
+      tags:
+      - Kadastraal Onroerende Zaken
+      description: |
+        Het zoeken van kadastraal onroerende zaken op basis van een burgerservicenummer.
+          1.  Ingeschreven persoon als zakelijk gerechtigde
+              -  burgerservicenummer (verplicht)
+              -  zakelijkGerechtigde__type (optioneel)
+
+        Met gebruik van de parameter expand kunnen zakelijkGerechtigden en privaatrechtelijkeBeperkingen direct worden meegeladen.
+
+        Het maximale aantal zoekresultaten dat geretourneerd wordt is aan de provider om te bepalen.
+        Als het resultaat van de de request dit aantal overtreft worden er geen resultaten geretourneerd en volgt er een foutmelding.
+      operationId: PostKadastraalOnroerendeZaken
```

Comment:
Wij zouden dit geen PostKadastraalOnroerendeZaken noemen maar ZoekKadastraalOnroerendeZakenMetBsnOfKadasterNatuurlijkPersoonIdentificatie. 
De verwerking van POST in de operationId verduidelijkt niet, maar is eerder verwarrend voor een aansluiter. Het gebruik van Zoek is dan communicatiever.

Reply:
Zal naamgeving aanpassen.